addon/utils/utils.js

```diff
@@ -4,9 +4,14 @@
  */
 
 import Ember from 'ember';
+import require from 'require';
 
 export function hasEmberData() {
-  return typeof self.DS !== 'undefined';
+  return require.has('ember-data');
```

Comment:
I'm getting an error here `utils.js Uncaught TypeError: Cannot read property 'has' of undefined` is there a version of ember this is incompatible with? I'm on ember 2.5